### PR TITLE
Swift 6 Toolchain

### PR DESCRIPTION
**Pull request by rikner**
Screenshot 2024-10-25 at 14 05 12

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -1,39 +1,54 @@
...
+
 readonly HOST=darwin-x86_64 # TODO: add more platforms
 
 copySwiftDependencyLibs() {
+    log "Copying Swift dependency libraries..."
+
     function copyLib {
         local DESTINATION="${LIBRARY_OUTPUT_DIRECTORY}/`basename "$1"`"
+        local LIB_NAME=`basename "$1"`
+        log "Copying ${LIB_NAME}"
```
Comment (rikner) [inline setup.sh:35]: [swift-android-toolchain] Copying Swift dependency libraries...
[swift-android-toolchain] Copying libGLESv2.so
[swift-android-toolchain] Copying libswiftAndroid.so
[swift-android-toolchain] Copying libandroid-spawn.so
[swift-android-toolchain] Copying libcamera2ndk.so
[swift-android-toolchain] Copying libmediandk.so
[swift-android-toolchain] Copying libstdc++.so
[swift-android-toolchain] Copying libssl.so
[swift-android-toolchain] Copying libOpenSLES.so
[swift-android-toolchain] Copying libGLESv3.so
[swift-android-toolchain] Copying libdispatch.so
[swift-android-toolchain] Copying libswiftRegexBuilder.so
[swift-android-toolchain] Copying libswift_Concurrency.so
[swift-android-toolchain] Copying libcrypto.so
[swift-android-toolchain] Copying libBlocksRuntime.so
[swift-android-toolchain] Copying libnghttp2.so
[swift-android-toolchain] Copying libssh2.so
[swift-android-toolchain] Copying libswiftCore.so
[swift-android-toolchain] Copying libnghttp3.so
[swift-android-toolchain] Copying libswiftSynchronization.so
[swift-android-toolchain] Copying libswift_StringProcessing.so
[swift-android-toolchain] Copying libjnigraphics.so
[swift-android-toolchain] Copying libiconv.so
[swift-android-toolchain] Copying libvulkan.so
[swift-android-toolchain] Copying libcharset.so
[swift-android-toolchain] Copying libm.so
[swift-android-toolchain] Copying libandroid.so
[swift-android-toolchain] Copying libc++_shared.so
[swift-android-toolchain] Copying libcares.so
[swift-android-toolchain] Copying libswiftDispatch.so
[swift-android-toolchain] Copying libswift_Builtin_float.so
[swift-android-toolchain] Copying liblog.so
[swift-android-toolchain] Copying liblzma.so
[swift-android-toolchain] Copying libz.so
[swift-android-toolchain] Copying libEGL.so
[swift-android-toolchain] Copying libGLESv1_CM.so
[swift-android-toolchain] Copying libOpenMAXAL.so
[swift-android-toolchain] Copying libz.so
[swift-android-toolchain] Copying libswift_RegexParser.so
[swift-android-toolchain] Copying libswift_math.so
[swift-android-toolchain] Copying libswiftObservation.so
[swift-android-toolchain] Copying libc.so
[swift-android-toolchain] Copying libdl.so
[swift-android-toolchain] Copying libc++.so
[swift-android-toolchain] Copying libc++_shared.so

Comment (ephemer) [reply setup.sh:33]: Hi @rikner that looks pretty good – I'm assuming it's after we already listed a number of EXCLUDED_LIBS in build.gradle?

To me it still looks at least half of them are not needed though. e.g. `libswiftObservation`, `libOpenMAXAL` `libswift_RegexParser` `libvulkan` `libjnigraphics` `libssh2` `nghttp3` `mediandk` `camera2ndk` `android-spawn` and probably more

Comment (ephemer) [reply setup.sh:33]: If we print it would also make sense to print the full path I think and not just the filename

**Review by ephemer — APPROVED** [review]
LGTM! I left a non-blocking comment about printing the full path if we're going to log which libs were copied

**Pull request merged**